src/block.ts

```diff
@@ -98,7 +98,7 @@ const parser = (docParser: DocParser) => (block: BlockElement): Result<string, B
             return new Ok({ kind: ElementType.RICH_LINK, url, linkText });
 
         case ElementType.TWEET:
-            return tweetContent(block.tweetTypeData.id, docParser(block.textTypeData.html))
+            return tweetContent(block.tweetTypeData.id, docParser(block.textTypeData?.html))
```

Comment:
Ah, I've actually fixed this on a separate branch. That change won't actually render the tweet, I think what you want is?

```suggestion
            return tweetContent(block.tweetTypeData.id, docParser(block.tweetTypeData.html))
```